website/docs/addon_usd_artist_contribution_workflow.md

```diff
@@ -0,0 +1,92 @@
+---
+id: addon_usd_artist_contribution_workflow
+title: USD Contribution Workflow
+sidebar_label: Contribution Workflow
+---
+
+:::info
+This page is still under construction 👷🚧🛠️🔜🏗️.
+:::
+
+# Ayon USD Contribution Workflow
+
+[A recent PR](https://github.com/ynput/ayon-core/pull/295) implemented a special **AYON USD Contribution Workflow**.
```

Comment:
After sometime, this PR will be no longer recent. 
So, could we tweak it a little, maybe 
```suggestion
**AYON USD Contribution Workflow** was introduced in this PR [Implement USD workflow with global asset/shot contributions plug-in](https://github.com/ynput/ayon-core/pull/295) 
```
It also make me wonder, should we change the mentioned PR's title https://github.com/ynput/ayon-core/pull/295. because the title mentions houdini, although it related to more than Houdini.